src/db/migrations/44-add-artist-fields.ts

```diff
@@ -0,0 +1,60 @@
+import { Knex } from 'knex';
+import _ from 'lodash';
+
+import { ethereumAddressFromId } from '../../utils/identifiers';
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.alterTable(Table.artists, table => {
+    table.string('address');
+    table.string('avatarUrl');
+    table.jsonb('externalLinks');
+    table.jsonb('theme');
+    table.jsonb('spinampLayoutConfig');
+  })
+  await updateViews(knex);
+
+  const artistUpdates = await knex.raw(`
+    select a.id, a.address, p."avatarUrl", p."createdAtBlockNumber" from raw_artists as a
+      inner join raw_artist_profiles as p
+      on a."id" = p."artistId"
+      where a."address" is null
+      and p."avatarUrl" is not null
+      order by p."createdAtBlockNumber" desc
+  `);
+
+  if (artistUpdates.rows.length > 0) {
+    const [withBlock, withoutBlock] = _.partition(artistUpdates.rows, row => !!row.createdAtBlockNumber);
+
+    const earliestDistinctUpdate = Object.values(
+      _.mapValues(
+        _.groupBy(withBlock, 'id'),
+        values => _.sortBy(values, 'createdAtBlockNumber')[0]
+      )
+    );
+
+    const updates = earliestDistinctUpdate.concat(withoutBlock).map((row: any) => {
+      return {
+        id: row.id,
+        address: ethereumAddressFromId(row.id),
+        avatarUrl: row.avatarUrl,
```

Comment:
let's make sure we check later potential profiles for the avatarUrl in case the first one is missing, eg, see how the frontend does it here: https://github.com/Snowfork/spinamp/blob/e2d315afb41f1a0b0b3ffca00f080d7aafe0f0d8/src/utils/artists.ts#L9